packages/@aws-cdk/toolkit-lib/lib/toolkit/toolkit.ts

```diff
@@ -153,6 +156,69 @@ export class Toolkit extends CloudAssemblySourceBuilder implements AsyncDisposab
     };
   }
 
+  /**
+   * Bootstrap Action
+   */
+  public async bootstrap(cx: ICloudAssemblySource, userEnvironmentSpecs: string[], options: BootstrapOptions = {}): Promise<void> {
+    const ioHost = withAction(this.ioHost, 'bootstrap');
+    const assembly = await this.assemblyFromSource(cx);
+
+    let environments;
+    if (userEnvironmentSpecs.length) {
+      environments = environmentsFromDescriptors(userEnvironmentSpecs);
+    } else {
+      // select all environments we can find, from stacks being bootstrapped
+      const stackCollection = assembly.selectStacksV2({
+        patterns: ['**'],
+        strategy: StackSelectionStrategy.ALL_STACKS,
+      });
+      environments = stackCollection.stackArtifacts.map(stack => stack.environment);
+    }
+
+    const source = options.source ?? BootstrapSource.default();
+
+    const parameters = options.parameters ?? BootstrappingParameters.default();
+
+    const bootstrapper = new Bootstrapper(source.render(), { ioHost: withAction(this.ioHost, 'bootstrap'), action: 'bootstrap' });
+
+    const sdkProvider = await this.sdkProvider('bootstrap');
+    const limit = pLimit(20);
+
+    // eslint-disable-next-line @cdklabs/promiseall-no-unbounded-parallelism
+    await Promise.all(environments.map((environment: cxapi.Environment) => limit(async () => {
+      await ioHost.notify(info(`${chalk.bold(environment.name)}: bootstrapping...`));
+      const bootstrapTimer = Timer.start();
+
+      try {
+        const { toolkitStackName } = this;
+        const bootstrapResult = await bootstrapper.bootstrapEnvironment(
+          environment,
+          sdkProvider,
+          {
+            ...options,
+            toolkitStackName,
+            source: source.render(),
+            parameters: parameters.render(),
+          },
+        );
+        const message = bootstrapResult.noOp
+          ? ` ✅  ${environment.name} (no changes)`
+          : ` ✅  ${environment.name}`;
+
+        await ioHost.notify(success('\n' + message));
```

Comment:
Please avoid the deprecated `success` helper. Instead use `info` and use `chalk.green` to make the message green.

Reply:
Will do.